Pod/Classes/ValueStepper.swift

```diff
@@ -124,6 +118,7 @@ private enum Button: Int {
     
     public required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
You're right. I thought it was needed to get IB to pick up the `intrinsicContentSize` but it's not. I'm removing it.